GOVERNANCE.md

```diff
@@ -28,8 +26,14 @@ No more than one-fourth of the CC members may be affiliated with the same employ
 
 The CC members shall consist of active members of Community Projects and the two Individual Membership Directors as defined in Section 10.
```

Comment:
We could, but I'm a little wary of getting the TSC and CommComm charter's out of sync, and I'm also not particularly up for getting the TSC charter updated. Updating charters is a serious pain, and I'd prefer consistency over accuracy at this point TBQH.